### PR TITLE
Add content import button

### PR DESCRIPTION
**Pull request by codiebeulaine**
## Purpose
_Add import button to import content from the CMS_

### CODE REVIEW

File: home/utils.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        else:
+            block = blocks.StructBlock(
+                [
+                    ("message", blocks.TextBlock()),
+                ]
+            )
+            block_value = block.to_python({"message": message["answer"]})
+            return block_value
+
+    # def create_tags(tags, page):
```
Comment (erikh360) [inline home/utils.py:46]: can probably remove this

File: home/utils.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    #     for tag in tags:
+    #         created_tag, _ = Tag.objects.get_or_create(name=tag)
+    #         page.tags.add(created_tag)
+
+    ContentPage.objects.filter(id__lte=1000).delete()
```
Comment (erikh360) [inline home/utils.py:51]: Why are we doing this in 3 lines?

File: home/utils.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    ContentPage.objects.filter(id__lte=2000).delete()
+    ContentPage.objects.filter(id__lte=3000).delete()
+    data = json.load(file)
+
+    for message in data["data"]:
+        try:
+            print(message)
```
Comment (erikh360) [inline home/utils.py:58]: print

File: home/views.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+from django.shortcuts import render
+from .forms import UploadFileForm
+from .utils import import_content
+
+
+def upload_file(request):
+    if request.method == "POST":
+        form = UploadFileForm(request.POST, request.FILES)
+        if form.is_valid():
+            job = handle_uploaded_file(request.FILES["file"])
```
Comment (erikh360) [inline home/views.py:11]: Can't we just call `import_content` directly here?

File: home/utils.py (hunk 2 above)
Comment (codiebeulaine) [reply home/utils.py:51]: wagtail delete only allows 1000 at a time

**Pull request merged**